Sage/carbon
# Enforce a body with post requests

# Description

This enforces when the TableAjax is used with a post request:
* It does not format the filter as it is not needed for a body.
* It sends the data instead of adding the filter as a query.

# Related Issues / Pull Requests

Extends: https://github.com/Sage/carbon/pull/2042

The above works for small data sets, but as this is designed to allow larger request payloads, superagent has `.send()` option to build a body separate to `.query()`.

# Testing Instructions

Confirm when doing a get request it builds the url.
Confirm when doing a post request it passes the query params into the body of the request.
